cmd/server/pactasrv/conv_oapi_to_pacta.go

```diff
@@ -0,0 +1,58 @@
+package pactasrv
+
+import (
+	"fmt"
+	"regexp"
+
+	api "github.com/RMI/pacta/openapi/pacta"
+	"github.com/RMI/pacta/pacta"
+)
+
+var initiativeIDRegex = regexp.MustCompile(`^[a-zA-Z0-9_-]+$`)
+
+func initiativeCreateToPACTA(i *api.InitiativeCreate) (*pacta.Initiative, error) {
```

Comment:
I think we've got enough overload on `PACTA` (the server, domain package, versions, etc), maybe we call the functions `fromOAPI` and `toOAPI` to indicate direction less ambiguously

Reply:
SGTM